environment.sh

```diff
@@ -108,12 +108,13 @@ else
           repo_info=(${line//@/ })
           repo=${repo_info[0]}
           repo_branch=${repo_info[2]}
-          last_update_time=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/ihmeuw/$repo/commits?sha=$repo_branch | jq '.[0].commit.committer.date')
+          last_update_time=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/ihmeuw/$repo/commits?sha=$repo_branch | jq .[0].commit.committer.date)
       else
           repo=$(echo "$line" | cut -d '>' -f1)
-          last_update_time=$(curl -s https://pypi.org/pypi/$repo/json | jq '.releases | to_entries | max_by(.key) | .value | .[0].upload_time')
+          last_update_time=$(curl -s https://pypi.org/pypi/$repo/json | jq -r '.releases | to_entries | max_by(.key) | .value | .[0].upload_time')
       fi
-      if [[ $creation_time > $last_commit_time ]]; then
+      last_update_time=$(date -d "$last_update_time" '+%Y-%m-%d %H:%M:%S')
+      if [[ $creation_time < $last_update_time ]]; then
```

Comment:
This update is from having to format the date to match the format of creation time.